invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/container.overrides

```diff
@@ -6,7 +6,6 @@
   background-color: @highlightHeaderColor;
   padding-top: @celledPadding;
   padding-bottom: @celledPadding;
-  margin-bottom: @defaultMargin;
```

Comment:
could you include screenshots of all the other affected places which are using this class>

Reply:
found a way without breaking anything :smile: